content/articles/data-viz-chartify/index.md

```diff
@@ -0,0 +1,365 @@
+
+- **Step 1**: Uninstall Chrome.
+- **Step 2**: Download Older Chrome Version. You can get this from sites like [FileHippo](https://www.slimjet.com/chrome/google-chrome-old-version.php) and [SlimJet](https://filehippo.com/download_google-chrome/history/) since there is an official repository hosting an older build of the Chrome browser.
+- **Step 3**:  Disable Chrome Auto-Updates. 
+
+You can get a more detailed step-by-step tutorial in this article, [How to Downgrade and Install Older Versions of Chrome?](https://browserhow.com/how-to-downgrade-and-install-older-version-of-chrome/).
+
+### Data Visualization using Chartify
+
+To get started with our charts plotting, type and run the below code snippet.
+
+```python
+ch = chartify.Chart()
+ch.show()
+```
+The code we just ran will give us the output below. 
+
+![Chartify](/engineering-education/data-viz-chartify/chartifyone.png)
+
+*Screenshot by author*
+
+Chartify makes it relatively easy for first-timers to get started. That's one amazing thing about it. This empty chart that got displayed shows users of the library how they can fill the chart with data. 
+
+Let’s visualize the Pokemon type distribution using a bar chart.
+
+### Pokemon type distribution
+
+A bar chart represents data by using bars of different heights. We would use on to see the distribution of the various Pokemon types. First, we need to create a data frame by grouping the Pokemon type 1 and its generation. This can be done using the code snippet below:
+
+```python
+bar_data = (pokemon_df.groupby('type1')[['generation']].sum()
+            .reset_index()
+           )
+bar_data
+```
+Next, we would visualize this data frame. 
+
+```python
+ch = chartify.Chart(blank_labels=True, x_axis_type='categorical')
+ch.set_title("Pokemon EDA using Chartify")
+ch.set_subtitle("Pokemon type distribution")
+ch.plot.bar(
+        data_frame=bar_data,
+        categorical_columns=['type1'],
+        numeric_column='generation')
+ch.show()
+```
+The first few arguments that we passed are `.set_title` and `.set_subtitle` which are for your chart title and subtitle. You can specify a name for your x and y axes using `.axes.set_xaxis_label` and `.axes.set_yaxis_label` respectively. The last argument, `.plot.bar` requires your data_frame which is the data to be plotted. The other parameters define the categorical and numerical columns you want to plot. 
+
+![Chartify](/engineering-education/data-viz-chartify/chartifytwo.png)
+
+*Screenshot by author*
+
+Next, let’s look at the pokemon type distribution but this time taking account of the ‘is_legendary’ attribute of each Pokemon. We would also start by creating a dataframe of the data needed. We can achieve this by grouping our Pokemon by the “type1” and “is_legendary” columns. This can be done by running the code snippet below:
```

Comment:
```suggestion
Next, let’s look at the Pokemon type distribution but this time taking account of the `is_legendary` attribute of each Pokemon. We would also start by creating a data-frame of the data needed. We can achieve this by grouping our Pokemon by the `type1` and `is_legendary` columns. This can be done by running the code snippet below:
```

Reply:
Use back ticks when using names included in the codes